packages/shorebird_cli/lib/src/commands/list/list_releases_command.dart

```diff
@@ -0,0 +1,96 @@
+import 'package:collection/collection.dart';
+import 'package:intl/intl.dart';
+import 'package:mason_logger/mason_logger.dart';
+import 'package:shorebird_cli/src/code_push_client_wrapper.dart';
+import 'package:shorebird_cli/src/commands/preview_command.dart';
+import 'package:shorebird_cli/src/config/config.dart';
+import 'package:shorebird_cli/src/logger.dart';
+import 'package:shorebird_cli/src/shorebird_command.dart';
+import 'package:shorebird_cli/src/shorebird_env.dart';
+import 'package:shorebird_code_push_client/shorebird_code_push_client.dart';
+
+/// {@template list_releases_command}
+/// A command to list available releases.
+/// {@endtemplate}
+class ListReleasesCommand extends ShorebirdCommand {
+  /// {@macro list_releases_command}
+  ListReleasesCommand() {
+    argParser
+      ..addOption(
+        'flavor',
+        help: 'The product flavor to use when building the app.',
+      )
+      ..addOption(
+        'limit',
+        help: 'Limit number of releases to be printed.',
+        defaultsTo: '$defaultLimit',
+      );
+  }
+
+  /// The default limit for the number of releases to be printed.
+  static const int defaultLimit = 10;
+
+  @override
+  String get description => 'List available releases.';
+
+  @override
+  String get name => 'list';
```

Comment:
I like the `list` name, but there are a lot of things we could list. I would create a `ReleasesCommand` and add this as a subcommand. See https://github.com/shorebirdtech/shorebird/blob/main/packages/shorebird_cli/lib/src/commands/patches/patches_command.dart for an example of this.

Reply:
Done 👍